clients/src/main/java/org/apache/kafka/common/Cluster.java

```diff
@@ -327,6 +346,14 @@ public Node controller() {
         return controller;
     }
 
+    public Collection<Uuid> topicIds() {
+        return topicIds.values();
+    }
+
+    public Uuid getTopicId(String topic) {
```

Comment:
nit: We don't use `get` prefix in other methods. Since this is part of the public API, can we change this to `topicId(String topic)` before merging?

Reply:
Thank you for your reminder, it's helpful to me.

Reply:
Can we also add this and any other new public APIs to the KIP page?

Reply:
Thank you for your reminder, I have added these 2 methods to the KIP page, take a look if you have time.

Reply:
Thanks. looks good!